index.yaml

```diff
@@ -9,6 +9,17 @@ indexes:
 # automatically uploaded to the admin console when you next deploy
 # your application using appcfg.py.
 
+- kind: ApiAuthAccess
+  properties:
+  - name: event_list
+  - name: owner
+  - name: expiration
+
+- kind: ApiAuthAccess
```

Comment:
I ran it starting from no changes and from only the first index. After both times, the file ended up like what was committed. So I presume that's what we need, since I don't really understand indexes fully either